File: Year-2023/LeetCode-August-Challenge/ExcelSheetColumnTitle.cpp
#include<bits/stdc++.h>
using namespace std;
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res = "";
        while(columnNumber){
            char ch = 'A' + (columnNumber - 1) % 26;
            res = ch + res;
            columnNumber = (columnNumber - 1) / 26;
        }
        return res;
    }
};

int main(){
    int columnNumber = 28;
    Solution *s = new Solution();
    cout<<s->convertToTitle(columnNumber)<<endl;
    return 0;
}

File: Year-2023/LeetCode-August-Challenge/ExcelSheetColumnTitle.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String res = "";
        while (columnNumber != 0) {
            char ch = (char) ('A' + (columnNumber - 1) % 26);
            res = ch + res;
            columnNumber = (columnNumber - 1) / 26;
        }
        return res;
    }
}

public class ExcelSheetColumnTitle {
    public static void main(String[] args) {
        int columnNumber = 28;
        System.out.println(new Solution().convertToTitle(columnNumber));
    }
}